x/cheqd/types/did_validation_test.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestIsDid(t *testing.T) {
+	cases := []struct {
+		valid bool
+		did   string
+	}{
+		{true, "did:cheqd:test:wyywywywyw"},
+		{true, "did:cheqd:test:wyywywywyw:sdadasda"},
```

Comment:
I'm unsure of how the rest of the validation works (haven't read ahead), but both of these values are incorrect

Reply:
Yes. PR in draft, Andrew is updating tests now.